### DISCUSSION TITLE
Recaptcha not showing after last update in combination with Cookiebot "necessary cookies only"

### DISCUSSION THREAD

**Discussion by mrwijsman** [solspace/craft-freeform#592] [Feature Requests]
Since the last update recaptcha wont show unless you accept all cookies when using Cookiebot, I think the cause of this issue is related to the latest patch 4.0.9 

"Updated to only load reCAPTCHA scripts when form(s) have reCAPTCHA enabled."

Cookiebot scans the website once a month and puts them in categopries (marketing, statistics etc) cookies showing up after the scan (or loaded on the fly) are being blocked.

- Craft Version: 4.3.6
- Freeform Version:4.0.9
- Freeform Edition: lite
- Fresh Install or Upgrade: upgrade

**Comment by kjmartens** [comment]
Sorry for the trouble @mrwijsman. I'll check with a developer and get back to you on this.

**Comment by kjmartens** [comment]
Sorry for the delay...

We've spent time looking at it... the reCAPTCHA cookie is a _Functional Cookie_, which Cookiebot blocks by default, and not a _Strictly Necessary Cookie_, which Cookiebot allows by default.

It appears that this is not something we could make work with the existing approach, but we could potentially add a setting that would take care of this. It appears to be an edge case though, so this is added to our list for future consideration. Alternatively, we do offer custom software development services where we can expedite the feature. Just let me know at support@solspace.com if you're interested. 🙂
